SPECS/shim/shim.spec

```diff
@@ -0,0 +1,33 @@
+%global debug_package %{nil}
+%global release_number 1
+Summary:        First stage UEFI bootloader
+Name:           shim
+Version:        15.4
+Release:        %{release_number}%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/rhboot/shim
+# This signed-shim tarball contains the shim binary signed with
+# the Microsoft UEFI CA key
+Source0:        signed-%{name}-%{version}-%{release_number}.tar.gz
+ExclusiveArch:  x86_64
```

Comment:
Just curious: why is there no shim for ARM64?

Reply:
Currently aarch64 shim 15.4 builds are in a bad state. They will break using binutils versions before 2.35, and even after that they may give unpredictable results. Due to this, aarch64 shims are not being accepted for signing at this time. I will update this package with an aarch64 shim when these issues are resolved and signing is allowed.

Reply:
Could you add the comment to the SPEC file to explain this?

Reply:
Added comment